packages/frontend-api/src/Resources/config/services.yml

```
@@ -5,7 +5,7 @@ services:
         public: false
 
     Shopsys\FrontendApiBundle\:
-        resource: '../../**/*{Resolver,Map}.php'
+        resource: '../../**/*{Facade,Map,Resolver,Mutation}.php'
```

```suggestion
        resource: '../../**/*{Facade,Map,Mutation,Resolver}.php'
```

I think it's nice to keep things alphabetically ordered, what do you think?

---

5be21b9f98706bd45ef8b5bbeda2e988435bca90